pkg/subctl/cmd/verify.go

```diff
@@ -0,0 +1,104 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/onsi/ginkgo/config"
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+
+	verifyconnectivity "github.com/submariner-io/submariner-operator/pkg/subctl/verify/connectivity"
+	verifyservicediscovery "github.com/submariner-io/submariner-operator/pkg/subctl/verify/servicediscovery"
+)
+
+var (
+	verifyConnectivity     bool
+	verifyServiceDiscovery bool
+	verifyAll              bool
+	//verboseConnectivityVerification bool
```

Comment:
Can we remove those comments? see my later comment about refactor.

Reply:
Done